src/util/Whamm.v3

```diff
@@ -302,14 +304,26 @@ component WhammVarBinder {
 						err.at(cv.parser.decoder).WhammProbeError("whamm probe", "nested function calls");
 						return null;
 					}
-					arg = bindCallResult(target, params, wi, op, expParams[i], cv, err);
+					// Call this function only if we have a match!
```

Comment:
This might be a bit shorter if you use a list of funexprs, so that you don't need to define an intermediate data structure.